### ISSUE TITLE
Pretrained models

### ISSUE THREAD

**Issue by abdelrahman-shaaban98** [leebebeto/Inter-Prototype#2]
Hello, 
if I need to use for example the Casia WebFace pretrained model, what model of the five models should I use?
<img width="516" alt="image" src="https://user-images.githubusercontent.com/52332496/168926486-96764244-bb2e-4341-b000-75617e114444.png">

I would understand if there is only two models, one for Casia and the other for MS1M, but there is 10 models, what's exactly the differenc ebetween them and why to chose one over the other?

Thanks in advance.

**Comment by YeolJ00** [comment]
Hello,

Thanks for your interest in our work. 
In our [paper](https://www.bmvc2021-virtualconference.com/assets/papers/0761.pdf), we evaluate our model on five different test datasets.
The pretrained models are the best performing models for each test set.
You can use any of the five links to verify the performance.

**Comment by abdelrahman-shaaban98** [comment]
Thank you for yout response.
But I'm a little confused, why there is one model for each evaluation set? Why not only one model train on Casia and used for all the 5 evaluation sets?
Is the evaluation process modify something in the model, so that we need to save one model for each evaluation set?

**Comment by YeolJ00** [comment]
The evaluation process is identicall for all test sets. 
So there is no problem for using any of the five models to evaluate all five tests sets. 
We just wanted to provide the best checkpoints from a single training procedure. 
Thus the accuracy obtained from the pretrained model may be slightly higher than the numbers reported in our paper.

**Comment by YeolJ00** [comment]
To avoid confusion, we now provide a single model for each dataset!

**Comment by abdelrahman-shaaban98** [comment]
OK. Thank you very much.
